source/blog/2018-07-10-Skydive-With-oVirt.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Skydive With oVirt 
+author: mburman
+tags: oVirt, oVirt 4.2, open source virtualization,network
+date: 2018-07-10 09:01:00 UTC
+---
+
+## Benefit to oVirt users
```

Comment:
The blog should begin by explaining what skydive is before going into the benefits. So I would begin with this sentence as an introduction:

So, "Skydive is an open source real-time network topology and protocols analyzer that provides a comprehensive way of understanding what is happening in your network infrastructure."

And then go into the Benefits to oVirt.

Reply:
It's what i did in the beginning and dominik asked to change the order. first say the benefits.
I don't mind

Reply:
Ok. Please change it back -- and sorry I didn't catch Dominik's comment or I would have disagreed sooner :)

Reply:
So first introduction and then benefits?

Reply:
Yes please :)

Generally always want to define a thing / explain what it is first, and then go into other stuff.

Reply:
done